pkg/drivers/kic/oci/volumes.go

```diff
@@ -30,6 +31,47 @@ import (
 	"k8s.io/klog/v2"
 )
 
+// RemoveVolume removes a volume
+func RemoveVolume(ociBin string, name string) error {
+	if !volumeExists(ociBin, name) {
+		return nil
+	}
+	rr, err := runCmd(exec.Command(ociBin, "volume", "rm", name))
```

Comment:
should we use -f here (force) ?

Reply:
Hmm, "Remove a volume by force, even if it is being used by a container"

I don't think the volume should be in use, we just deleted the container ?